mne/viz/misc.py

```diff
@@ -109,17 +109,25 @@ def plot_cov(cov, info, exclude=[], colorbar=True, proj=False, show_svd=True,
     import matplotlib.pyplot as plt
 
     fig_cov, axes = plt.subplots(1, len(idx_names), squeeze=False,
-                                 figsize=(2.5 * len(idx_names), 2.7))
+                                 figsize=(3.8 * len(idx_names), 3.7))
     for k, (idx, name, _, _) in enumerate(idx_names):
-        axes[0, k].imshow(C[idx][:, idx], interpolation="nearest",
-                          cmap='RdBu_r')
+        im = axes[0, k].imshow(C[idx][:, idx], interpolation="nearest",
+                               cmap='RdBu_r')
```

Comment:
you make me realize that white will not be zero in the plots. Can you symmetrize the colormap to it's between z = np.max(np.abs(C)) and -z

Reply:
sure

Reply:
The result of symmetrization:

![covariance_plot_colorbar_symmetrize](https://user-images.githubusercontent.com/18072563/36315489-cb119818-132f-11e8-8a20-4a918ab2df7b.png)